selfdrive/car/fw_versions.py

```diff
@@ -339,12 +339,17 @@ def match_fw_to_car_exact(fw_versions_dict):
       if ecu_type not in ESSENTIAL_ECUS and not len(found_versions):
         continue
 
+      # Ignore ecus in the database for data collection
```

Comment:
doesn't have to be in this PR or anything, but we should think about ways to make this more explicit

Reply:
yeah, was thinking the same thing

Reply:
+1 I would like to use something like this to gather/prep for adding ABS/ESP and combinationMeter for VW.